conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/ConjureJerseyFeature.java

```diff
@@ -16,30 +16,41 @@
 
 package com.palantir.conjure.java.server.jersey;
 
+import com.codahale.metrics.Meter;
 import com.fasterxml.jackson.jaxrs.cbor.JacksonCBORProvider;
 import com.palantir.conjure.java.serialization.ObjectMappers;
 import com.palantir.tracing.jersey.TraceEnrichingFilter;
+import com.palantir.tritium.metrics.registry.DefaultTaggedMetricRegistry;
+import com.palantir.tritium.metrics.registry.TaggedMetricRegistry;
 import javax.ws.rs.core.Feature;
 import javax.ws.rs.core.FeatureContext;
 
 public enum ConjureJerseyFeature implements Feature {
     INSTANCE;
 
+    private final Meter internalExceptionMeter;
+
+    ConjureJerseyFeature() {
+        TaggedMetricRegistry registry = DefaultTaggedMetricRegistry.getDefault();
```

Comment:
I think this guy is actually deprecated... would you mind flipping to the SharedTaggedMetricRgistry thing? https://github.com/palantir/tritium/blob/18e2697fd596e717129f455abbfccfe34780ad1d/tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/DefaultTaggedMetricRegistry.java#L34

Reply:
Are we considering that 'less' deprecated? https://github.com/palantir/tritium/pull/519

Reply:
I used `DefaultTaggedMetricRegistry.getDefault()` since it's used that way here: https://github.com/palantir/conjure-java-runtime/blob/develop/client-config/src/main/java/com/palantir/conjure/java/client/config/ClientConfigurations.java#L91

Is just instantiating a `new DefaultTaggedMetricRegistry()` the right thing to do? It's done that way here: https://github.com/palantir/conjure-java-runtime/blob/develop/okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/DispatcherMetricSet.java#L34

Reply:
I think the thing you've done here is OK, because ultimately the shared one _will_ get spat out into a `var/log/metric.log` file by sls-logging.  If you'd just created a brand `new DefaultTaggedMetricRegistry()` without hooking it up then the metrics wouldn't get logged.

Reply:
If we were writing this stuff from scratch, then wiring in an instance as a constructor param would be preferrable, but I think avoiding the java break is more important in this repo.